andela/Ah-frontend-xmen
# 164328865 Implement bug fixes from feedback

#### What does this PR do?
- Implements feedback received during the demo.

#### Description the task to be implemented
- reduce icon size for social buttons.
- provide a link to login on signup
- remove nonworking button on the editor (Calvin)
- get rid of resizing on the biofield(profile edit)
- registration modal should have signup not sign in
- reg modal should have a link to sign in
- reduce space between signup and social icons
- forgot password link was misplaced
- get rid of TX on create article editor
- introduce tooltips on icons
- reduce icons on article detail
- move comment actions to the same line

- show toast after closing the modal.
- replace value attribute with a placeholder(profile edit)

#### How can this be manually tested?
- Fetch this branch and check into it by running `git fetch origin bg-fixes-ui-164328865 && git checkout bg-fixes-ui-164328865`
- Install dependencies and start the local server by running `yarn install && yarn start`
- Log in with either `social auth` or the `login form`
- view changes

#### What are the related pivotal stories?
[#164328865](https://www.pivotaltracker.com/story/show/164328865)
